### PR TITLE
Wabco features

### PR DESCRIPTION
**Pull request by JelleBoersma**
Fixes #<issue>

## Checklist:
- [ ] My code follows the pykechain style
    - [ ] I added type hinting to all functions
    - [ ] I added documentation for all public functions
    - [ ] I locally used `tox` to check for dists and docs errors
    - [ ] My code passes the `pep8` and `flake8` linting checks and no warnings
    - [ ] I removed unused imports, using `Code > Optimize Imports` in PyCharm
    - [ ] I used [`black`](https://github.com/psf/black) to format the new code
- [ ] I have added tests that prove my fix is effective or that my feature works
    - [ ] I committed fresh test cassettes
    - [ ] I have proper test coverage (don't decline the coverage of the test)
- [ ] I asked another teammate to review the code
- [ ] I update the Changelog.md with the appropriate changes.

### CODE REVIEW

**Review by raduiordache — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,13 +4,25 @@ Change Log
...
+* :+1: Added `get_prefilters()` and `get_excluded_propmodel_ids()` methods to the `MultiReferenceProperty` class.
+* :+1: Added `alignment` keyword arguments for the creation of `ServiceCardWidget` and `ServiceWidget` classes.
+
+Backwards incompatible changes
+------------------------------
+
+The following changes are not compatible with previous functionality:
+
+* The `overwrite` keyword argument in the `set_prefilters()` method of the `MultiReferenceProperty` now only overwrites prefilters if explicitly provided with new ones. Removing all prefilters is now supported with the keyword argument `clear`, also a boolean.
+* Specifying prefilters via separate lists of properties, values and filter types is planned to be deprecated in January 2021 in favor of using `PropertyValueFilter` objects as input.
```
Comment (jberends) [inline CHANGELOG.rst:24]: nice

File: CHANGELOG.rst [hunk 2]
```diff
@@ -4,13 +4,25 @@ Change Log
...
+* :+1: Created `PropertyValueFilter` class to manage (pre)filters of `MultiReferenceProperty` and `FilteredGridWidget` objects.
+* :+1: Added `get_prefilters()` and `get_excluded_propmodel_ids()` methods to the `MultiReferenceProperty` class.
+* :+1: Added `alignment` keyword arguments for the creation of `ServiceCardWidget` and `ServiceWidget` classes.
+
+Backwards incompatible changes
+------------------------------
+
+The following changes are not compatible with previous functionality:
+
+* The `overwrite` keyword argument in the `set_prefilters()` method of the `MultiReferenceProperty` now only overwrites prefilters if explicitly provided with new ones. Removing all prefilters is now supported with the keyword argument `clear`, also a boolean.
```
Comment (jberends) [inline CHANGELOG.rst:23]: good point

File: pykechain/models/widgets/helpers.py [hunk 3]
```diff
@@ -354,57 +356,42 @@ def _initiate_meta(kwargs, activity, ignores=()):
...
+        if any(len(lst) != len(property_models) for lst in [values, filters_type]):
+            raise IllegalArgumentError('The lists of "property_models", "values" and "filters_type" should be the '
+                                       'same length.')
+        prefilters = [PropertyValueFilter(
+            property_model=pf[0],
+            value=pf[1],
+            filter_type=pf[2],
+        ) for pf in zip(property_models, values, filters_type)]
+
+        warnings.warn(
```
Comment (jberends) [inline pykechain/models/widgets/helpers.py:383]: I like the warning!

**Review by jberends — APPROVED** [review]
good improvements

**Pull request merged**